top100.php

```diff
@@ -192,6 +192,7 @@ function top100($realmid, &$sqlr, &$sqlc)
                                 <td>'.char_get_level_color($char['level']).'</td>';
         if ($type === 'level')
         {
+            // @TODO - fix N+ issue - query inside while loop
             $guild_name = $sqlc->result($sqlc->query('SELECT BINARY name AS name FROM guild WHERE guildid = '.$char['gname'].''), 0);
```

Comment:
N+1 query issue here, probably shouldn't be inside the loop. Need to eager load this information elsewhere.